packages/react-components/theme-designer/src/components/Sidebar/UseTab.tsx

```diff
@@ -49,7 +48,8 @@ export interface UseTabProps {
 export const UseTab: React.FC<UseTabProps> = props => {
   const styles = useStyles();
 
-  const { theme, setTheme, dispatchAppState, sidebarId, formState, setFormState, isDark, setIsDark } = props;
+  const { theme, setTheme, sidebarId, formState, setFormState, isDark, setIsDark } = props;
```

Comment:
_nit:_ Order alphabetically

```suggestion
  const { formState, isDark, setFormState, setIsDark, setTheme, sidebarId, theme } = props;
```